packages/blitz-rpc/src/loader-server.ts

```diff
@@ -56,21 +53,26 @@ export async function transformBlitzRpcServer(
   let code = blitzImport + src
   code += "\n\n"
   for (let resolverFilePath of resolvers) {
-    const relativeResolverPath = slash(relative(dirname(id), join(root, resolverFilePath)))
     const routePath = convertPageFilePathToRoutePath(resolverFilePath, options?.resolverPath)
-    code += `__internal_addBlitzRpcResolver('${routePath}',() => import('${relativeResolverPath}'));`
+
+    code += `__internal_addBlitzRpcResolver('${routePath}',() => import('${resolverFilePath}'));`
     code += "\n"
   }
   // console.log("NEW CODE", code)
   return code
 }
 
-export function collectResolvers(directory: string, pageExtensions: string[]): Promise<string[]> {
-  return recursiveFindResolvers(directory, buildPageExtensionRegex(pageExtensions))
+export function collectResolvers(
+  directory: string,
+  rpcFolders: string[],
+  pageExtensions: string[],
+): Promise<string[]> {
+  return recursiveFindResolvers(directory, rpcFolders, buildPageExtensionRegex(pageExtensions))
 }
 
 export async function recursiveFindResolvers(
   dir: string,
+  rpcFolders: string[],
```

Comment:
move this to the last argument so it can be optional because we only need it at the top level recursion

Reply:
As a consequence of moving the param rpcFolders last.  I've had to change the execution of recursiveFindResolvers to the following.

> return recursiveFindResolvers(directory, buildPageExtensionRegex(pageExtensions), undefined, [], directory, rpcFolders)

Is this ok?